pyproject.toml

```diff
@@ -45,6 +45,7 @@ test = [
     "pytest-mock",
     "pytest-subprocess",
     "responses",
+    "toml"
```

Comment:
You could always just bump to 3.11 :smile:

Reply:
the stdlib tomllib is read only, not write 🤷